Sway workspace styles don't get removed consistently

The sway workspace styles (`.solo`, `.<appid>`, `.empty`, etc) are sometimes not removed anymore.

## (Preliminary) Steps to reproduce

I did not test this with a mostly empty config yet.

-  Add a style for `.firefox`, e.g.

```
window#waybar.firefox {
  background-color: #0088ff;
}
```

- Go to a new workspace
- Open two (tiled) windows (any application)
- Go to a new workspace
- Open firefox (the style should now be applied)
- Go back to the workspace with the two windows

## Expected Behavior

The style applied for `.firefox` is removed again

## Actual Behavior

The style applied for `.firefox` is still present

## Further info

This is a regression in `0.9.18`. `0.9.17` did not have this bug.

I wasn't very sure about the `.solo` and `.empty` styles not working. Could be that the `GTK_DEBUG=interactive` just didn't update properly.

I assume your fix is about what I reported. I tried to test your fix, but I can't get waybar to compile: `ninja -C build` is reporting some problems in `Catch2` (`catch_string_manip.hpp:47:14: error: ‘uint64_t’ in namespace ‘std’ does not name a type; did you mean ‘wint_t’?`). If its okay I'll just wait for the next release.

--- a/src/modules/sway/window.cpp
+++ b/src/modules/sway/window.cpp
@@ -178,10 +178,6 @@ auto Window::update() -> void {
     } else {
       mode += 32;
     }
-    if (!app_id_.empty() && !bar_.window.get_style_context()->has_class(app_id_)) {
-      bar_.window.get_style_context()->add_class(app_id_);
-      old_app_id_ = app_id_;
-    }
   }
 
   if (!old_app_id_.empty() && ((mode & 2) == 0 || old_app_id_ != app_id_) &&

